source/find_getmore_killcursors_commands.rst

```diff
@@ -294,23 +316,56 @@ when sent to a secondary. The OP_QUERY namespace MUST be the same as for the
 
 .. _getMore: https://www.mongodb.com/docs/manual/reference/command/getMore/
 
-The **batchSize** option of **getMore** command MUST be an int32 larger than 0. If **batchSize** is equal to 0 it must be omitted. If **batchSize** is less than 0 it must be turned into a positive integer using **Math.abs** or equivalent function in your language.
+.. code:: typescript
+
+  interface GetMoreCommand {
+    /** Set to the nonzero cursor id */
+    getMore: int64;
+    /** Set to the namespace returned by the initial command response */
+    collection: string;
+    /**
+     * User configurable document count for the batch returned for this getMore.
+     * Only attached to command document if nonzero.
+     */
+    batchSize?: PositiveIntegerNumber;
+    /**
+     * User configurable time limit enforced by the server.
+     */
+    maxTimeMS?: PositiveIntegerNumber;
+    /**
+     * User configurable comment that can be used to identify the operation in logs.
+     * This can be any BSON value.
+     */
+    comment?: BSONValue;
+  }
+
+The **batchSize** option of **getMore** command MUST be an unsigned int larger than 0. If **batchSize** is equal to 0 it must be omitted. If **batchSize** is less than 0 it must be turned into a positive integer using **Math.abs** or equivalent function in your language.
```

Comment:
Is this paragraph redundant in light of using "PositiveIntegerNumber" above?

I traced this text back to [1aabad4](https://github.com/mongodb/specifications/commit/1aabad462d40108e759b8e508f3639bb579bd2dd#diff-03c14c04c35bbec5ed4b9d5d4d75aa64c9d4097de906267d849c6a306090d6ceR520) from 7.5 years ago. AFAIK, it only pertained to [Mapping OP_QUERY behavior to the find command limit and batchSize fields](https://github.com/mongodb/specifications/blob/master/source/find_getmore_killcursors_commands.rst#mapping-op-query-behavior-to-the-find-command-limit-and-batchsize-fields), which is still discussed elsewhere in the spec.

There should be no concern with removing this. /cc @alcaeus